tools/replay/can_replay.py

```diff
@@ -83,7 +83,7 @@ def connect():
 
 if __name__ == "__main__":
   if PandaJungle is None:
-    print("\33[31m", "WARNING: cannot connect to jungles. Clone the jungle library to enable support:", "\033[0m")  # type: ignore
```

Comment:
I added this on my computer because it said the print was unreachable. Probably because the `panda_jungle` import actually succeeds for me.

Reply:
Bool seems to fix a problem though I've just made a panda_jungle.py file instead of cloning the whole repo.

Reply:
I'd give it an `Optional[PandaJungle]` type, but PandaJungle doesn't really exist if it can't be imported

Reply:
Tried to think of something clean too, but couldn't make mypy happy. Guess it's kind of a weird pattern.

I'm ok with this bool since it's just some tools file.